Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataDocumentBean.java

```diff
@@ -264,23 +274,24 @@ public String getPatientId() {
         return getDocumentForm().getPatientId();
     }
 
-    public void setPatientId(String patientId){
-        getDocumentForm().setPatientId(patientId);
-        getDocumentForm().setSourcePatientId(patientId);
+    public void setPatientId(String patientIdentifierIso) {
+        getDocumentForm().setPatientId(patientIdentifierIso);
+        getDocumentForm().setSourcePatientId(patientIdentifierIso);
```

Comment:
removed Line: 278 and 279 - this already being done in the HelperUtil.updateDocumentBy